javascripts/adaptive-placeholder.js

```diff
@@ -0,0 +1,20 @@
+app.adaptivePlaceholder = {
+
+  changeFilledState: function () {
+    var input = $(this);
+    var value = input.val();
+    
+    if(value != '' && value != undefined) {
```

Comment:
Mind adding whitespace after the `if` please? We really should set up a standard for this and sass files.